### PR TITLE
Project bootstrapping

### PR DESCRIPTION
**Pull request by Chroq**
*Trello* : https://trello.com/c/scZW2kOm/1-1-as-norbert-i-want-to-be-able-to-see-a-particular-state-of-a-hex-game

Minimal bootstrapping : 

- [x] Makefile
- [x] Basic entry point
- [x] Tests 

## Expected results

make start
```
Start the project
python3 src/main.py
Hex
```

make test
```
Launch tests
python3 -m unittest /home/ubuntu/PycharmProjects/hex-python/src/main_test.py
.
----------------------------------------------------------------------
Ran 1 test in 0.000s

OK
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (zyhou) [inline .gitignore]: You can't exclude all IDE in your gitignore. Exclude this file in you local git in `.git/info/exclude` file

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.PHONY: default install start test
```
Comment (zyhou) [inline Makefile]: I didn't see any `install` command

File: src/main.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+    def __init__(self):
+        self.grid = []
+
+
+class Stone:
+
+    state = 0
+
+###
```
Comment (zyhou) [inline src/main.py]: useless comments ?

File: config/dummy.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+[
```
Comment (zyhou) [inline config/dummy.json]: useless file?

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+.DEFAULT_GOAL := help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+start: ## Start project
+	@echo "Start the project"
+	python3 src/main.py
```
Comment (zyhou) [inline Makefile]: I don't have python on my machine, how I can start the project ?

Comment (Luwangel) [reply Makefile]: https://www.python.org/downloads/ :smile:

**Pull request merged**